dashboard/fixtures/certificates.json

```diff
@@ -5,14 +5,14 @@
 	    "certificate_type": "verified",
 	    "status": "downloadable",
 	    "download_url": "http://www.example.com/demo.pdf",
-	    "grade": "0.98"
+	    "grade": 98
```

Comment:
It looks like we are still passing in this JSON in the `Certificates` objects in our tests like we did before. Wouldn't this cause a difference of behavior between our tests and the edX API?

If we need `grade` to be an int in the Elasticsearch index for some reason, we should make that change in the serializer function in `search/api.py`. But I also still don't understand why grade needs to be an int here anyway?